client/blocks/login/lost-password-form.jsx

```diff
@@ -27,6 +30,19 @@ const LostPasswordForm = ( {
 		}
 	};
 
+	const getAuthAccountTypeRequest = async ( emailAddress ) => {
+		const resp = await window.fetch(
```

Comment:
I found this existing code that's similar: https://github.com/Automattic/wp-calypso/blob/bef9cea64627aa28bd50edb0cb551ec659162551/client/state/data-layer/wpcom/users/auth-options/index.js#L13-L22

Can we reuse it?

Reply:
@ilyasfoo Good catch. That was my initial thought as well. However, the issue with that `getAuthAccountType` is that it modifies the state rather than returning a response. Given that we're in the `onClick` method, checking the state becomes challenging unless we restructure the code.

Reply:
Ah ok, makes sense